src/scss/pages/_degree-search.scss

```diff
@@ -0,0 +1,59 @@
+.degree-title-wrap {
+  box-sizing: border-box;
+  display: table;
+  font-size: 17px;
+  margin-bottom: 0;
+  padding: 15px;
+  width: 100%;
+
+  &:hover,
+  &:active,
+  &:focus {
+    text-decoration: none;
+  }
+
+  @include media-breakpoint-up( xs ) {
```

Comment:
Did you mean media-breakpoint-up(sm) here?  If not, remove this media query and just update the font size on line 4